docs/nodes/nodes-library/nodes/Medium/README.md

```diff
@@ -0,0 +1,38 @@
+---
+permalink: /nodes/n8n-nodes-base.medium
+---
+
+# Medium
+
+[Medium](https://www.medium.com/) is an online publishing platform.
+
+::: tip 🔑 Credentials
+You can find authentication information for this node [here](../../../credentials/Medium/README.md).
+:::
+
+## Basic Operations
+
+- Publication
+	- Get all publications
+- Post
+	- Create a post
+
+## Example Usage
+
+This workflow allows you to get all publications from Medium. You can also find the [workflow](https://n8n.io/workflows/593) on the website. This example usage workflow uses the following two nodes.
```

Comment:
This isn't a super helpful example. I think more useful example here would be creating a post and adding it to a publication.